R/sem.R

```diff
@@ -487,12 +491,15 @@ checkLavaanModel <- function(model, availableVars) {
   }
 
  # add missing value removal footnote
- if(options$missing == "listwise"){
-   nrm <- nrow(dataset) - lavaan::lavInspect(semResults[[1]], "ntotal")
-   missingFootnote <- gettextf("A total of %g cases were removed due to missing values. You can avoid this by choosing 'FIML' under 'Missing Data Handling' in the Estimation options.",
-                               nrm)
-   fittab$addFootnote(message = missingFootnote)
- }
+  if(options$missing == "listwise") {
+    nrm <- nrow(dataset) - lavaan::lavInspect(semResults[[1]], "ntotal")
+    if (nrm != 0) {
+      missingFootnote <- gettextf("A total of %g cases were removed due to missing values. You can avoid this by choosing 'FIML' under 'Missing Data Handling' in the Estimation options.",
```

Comment:
this is probably fine, but `%s` would be probably sufficient, or perhaps `%i` if you want to insist that it's an integer
```suggestion
      missingFootnote <- gettextf("A total of %g cases were removed due to missing values. You can avoid this by choosing 'FIML' under 'Missing Data Handling' in the Estimation options.",
```